src/main/java/racingcar/service/RoundRaceService.java

```diff
@@ -0,0 +1,40 @@
+package racingcar.service;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import racingcar.model.CarName;
+import racingcar.model.CarScores;
+import racingcar.model.Distance;
+import racingcar.view.OutputView;
+
+public class RoundRaceService {
+    private static final int MIN_RANDOM_RANGE = 0;
+    private static final int MAX_RANDOM_RANGE = 9;
+    private static final int MIN_MOVE_DISTANCE = 4;
+    private static final int MAX_MOVE_DISTANCE = 9;
+
+    public static CarScores playGame(CarScores previousCarScores) {
+        LinkedHashMap<CarName, Distance> nowScores = new LinkedHashMap<>();
```

Comment:
HashMap 사용이 인상적이네요!👍

Reply:
```suggestion
        Map<CarName, Distance> nowScores = new LinkedHashMap<>();
```
다형성을 고려해 이렇게 수정하는 건 어떤가요?

Reply:
@jhyoo1203 다형성은 하나의 클래스에 대해 여러 자식 클래스가 있을 때, 해당 자식 클래스들을 하나로 관리하고 싶을 때 사용하면 좋을 거라고 생각합니다! 맵과 링크드해시맵도 상속 관계이기는 하지만, 제가 작성한 프로그램에서는 두 컬렉션의 상속 관계를 이용한 것 같진 않아요. 만약 CarName과 Distance의 자식 클래스가 있는 경우엔 재현님 말씀처럼 다형성을 고려하면 좋을 것 같은데 이 부분은 어떻게 생각하시는지 궁금합니다!

Reply:
@kaswhy Map 인터페이스를 사용하여 선언하는 것을 제안한 이유는 이후에 다른 Map 구현체, 예를 들어 HashMap이나 TreeMap으로 변경할 때 쉽게 변경할 수 있는 유연성이 생기기 때문입니다! 코드의 유지보수성과 확장성 측면에서 좋을 것 같아 사소하지만 적용해보면 어떨까 생각해서 제안드렸습니다!